sunpy/tests/figure_hashes_mpl_dev_ft_261_astropy_dev.json

```diff
@@ -28,7 +28,8 @@
   "sunpy.map.tests.test_plotting.test_plot_aia171_superpixel_nowcsaxes": "d516c9d6a28fecdf0bde1e9a0c4663fd625a2bfb9e91f158cd1c6ed0eb06e906",
   "sunpy.map.tests.test_plotting.test_plot_masked_aia171_superpixel": "154586d5635f895b5f9170353f70f427e23ab0be4ccabf743e668864653f54e3",
   "sunpy.map.tests.test_plotting.test_plot_masked_aia171_superpixel_nowcsaxes": "ec4cfe2687ea0a1f62f1a80f07e69ef5a89ef5aa2dbebc58599fb16b98ebafcb",
-  "sunpy.map.tests.test_plotting.test_draw_contours_aia": "b78457a0d2ea6ab847540e73e7a266a5cb75dc12fc74222d6a75330bd70ee26c",
+  "sunpy.map.tests.test_plotting.test_draw_contours_aia": "aaa96e3e9b89df02ccf7af0aecfcdbd5a8b2b70aa56693aa57056c8f7a2d1d22",
```

Comment:
For the record, I had to update the hash for this figure test because one pixel changed.  <shrug>